hub/core/chunk_engine/tests/common.py

```diff
@@ -119,9 +107,8 @@ def assert_chunk_sizes(
 def run_engine_test(
     arrays: List[np.ndarray], storage: StorageProvider, batched: bool, chunk_size: int
 ):
-    clear_if_memory_provider(storage)
-
-    key = random_key()
+    storage.clear()
+    key = current_test_name(with_uuid=True)
```

Comment:
I think it would be much easier if we gave the storage provider with a different root automatically so we can use the same keys each time

Reply:
yep you're absolutely right. just did this. now we don't define keys within test bodies but rather when the fixture is created (scope=function now instead of session) it will auto do this.